src/frontend/src/flows/addDevice/manage/pollForTentativeDevice.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
- You don't save all the JSON file in a same directory?
- Should this directory or filename be prefixed with the lang?

like `/src/i18n/en/...`?

Reply:
If you have all files in a same folder it is easier for contributors or translators to find which files to translate.

Reply:
Sure language within the json sounds good 👍